sdk-extensions/incubator/src/test/java/io/opentelemetry/sdk/extension/incubator/fileconfig/component/ResourceFirstComponentProvider.java

```diff
@@ -6,28 +6,22 @@
 package io.opentelemetry.sdk.extension.incubator.fileconfig.component;
 
 import io.opentelemetry.api.incubator.config.DeclarativeConfigProperties;
-import io.opentelemetry.sdk.autoconfigure.spi.Ordered;
 import io.opentelemetry.sdk.autoconfigure.spi.internal.ComponentProvider;
 import io.opentelemetry.sdk.resources.Resource;
 
-public class ResourceOrderedFirstComponentProvider implements ComponentProvider<Resource>, Ordered {
+public class ResourceFirstComponentProvider implements ComponentProvider<Resource> {
```

Comment:
why is this not ordered any more?

Reply:
Resource detectors are now modeled like other SDK extension points. That is, they are referred to by a name and YAML object node with optional configuration parameters. Additionally, their names are referenced objects in an array, so the data model now explicitly orders them.

E.g.:
```
resource:
  attributes:
    - name: service.name
      value: foo
  detection/development:
    detectors:
      - os:
      - host:
      - foo:
```

PR here: https://github.com/open-telemetry/opentelemetry-configuration/pull/188